chance/p5js/Popup-Circles/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: p5js
+author: Daniele Maselli
+author-url: https://github.com/DanieleMaselli 
+title: Popup Circles 
+tags: /chance/repeat/Law of Similarity/random
```

Comment:
tags are still wrong. Please take a look at https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#markdown-and-front-matter